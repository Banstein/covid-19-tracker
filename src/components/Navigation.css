.navbar {
  padding: 5px;
}

i {
  padding: 0 5px;
}
