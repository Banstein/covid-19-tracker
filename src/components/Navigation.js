import React from 'react';
import { NavLink } from 'react-router-dom';
import './Navigation.css';

function Navigation() {
  return (
    <header className="navbar bg-secondary">
      <ul className="flex justify-between container">
        <li>
          <NavLink to="/">
            <i className="fa-solid fa-angle-left" />
          </NavLink>
        </li>
        <li>
          <NavLink to="/">
            Covid-19 Tracker
          </NavLink>
        </li>
        <li>
          <i className="fa-solid fa-microphone" />
          <i className="fa-solid fa-gear" />
        </li>
      </ul>
    </header>
  );
}

export default Navigation;
