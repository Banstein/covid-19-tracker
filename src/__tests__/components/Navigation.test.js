import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import store from '../../__mocks__/configureStore';
import Navigation from '../../components/Navigation';

describe('CNavigation snapshot', () => {
  it('renders correctly', () => {
    render(
      <Provider store={store}>
        <BrowserRouter>
          <Navigation />
        </BrowserRouter>
      </Provider>,
    );
  });
});
