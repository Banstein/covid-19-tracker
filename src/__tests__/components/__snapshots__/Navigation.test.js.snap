// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CNavigation snapshot renders correctly 1`] = `
<header
  className="navbar bg-secondary"
>
  <ul
    className="flex justify-between container"
  >
    <li>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        <i
          className="fa-solid fa-angle-left"
        />
      </a>
    </li>
    <li>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Covid-19 Tracker
      </a>
    </li>
    <li>
      <i
        className="fa-solid fa-microphone"
      />
      <i
        className="fa-solid fa-gear"
      />
    </li>
  </ul>
</header>
`;
